### DISCUSSION TITLE
Sync with Git:  Cannot read property permissions of "undefined"

### DISCUSSION THREAD

**Discussion by Zatryc98** [requarks/wiki#3395] [Help / Questions]
OS: Debian 9 (on both instances) 
Wiki.js version: 2.4.107 (on both instances) 
Database engine: Postgresql 12.3  (on both instances)

Hello, i am working with two instances,both synchronized via git,with the same URI repository and both in bi-directional mode.
When i create or edit a page, everything works correctly and the user is recognized as "Administrator".

After doing some tests, I noticed that when you delete a page via wikijs on machine A, synchronization does not take place, as machine B does not delete the page. The error is: "Cannot read property 'permissions' of undefined".
This is because when you delete a page, the user on Git is not recognized as "Administrator", but precisely as "undefined".

The question is, am i doing something wrong or it is just a bug?

Thank you

**Comment by tionis** [comment]
I also have this error in my case a sync between local notes and notes on the wiki server. The sync also fails, or at least throws an error, if theres an ":" in the title (yaml-header) of markdown
